frontend/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import AddIcon from '@mui/icons-material/Add';
+import HistoryIcon from '@mui/icons-material/History';
+import { useNavigate } from "react-router-dom";
+
+export default function Dashboard() {
+    return (
+        <div className="page-container">
+            <div className="sidebar">
+                <Logo />
+            </div>
+            <div className="main-content">
+                <MainContent />
+            </div>
+        </div>
+
+    );
+
+    function Logo() {
+        return (
+            <div className="sidebar-logo-container">
+            <span className="sidebar-logo-text-bold">
+            FlowLeaflets <br />
+            </span>
+            <span className="sidebar-logo-text-small">Clinical Logbooks</span>
+            </div>
+        );
+    }
```

Comment:
This looks quite similar to `Logo `defined in `Navbar.jsx`. Depending on what our plan is for this, could we consider pulling it into a separate, reusable component that can be used across the application? If we just plan to replace this with a photo then ignore my comment 😆